bucket/windows-terminal.json

```diff
@@ -3,6 +3,7 @@
     "description": "The new Windows Terminal, and the original Windows console host - all in the same place!",
     "homepage": "https://github.com/microsoft/terminal",
     "license": "MIT",
+    "notes": "Add Windows Terminal as a context menu option by running as an administrator `reg import \"$dir\\install-context.reg\"`",
```

Comment:
Admin rights are not required for HKEY_CURRENT_USER, right?

Reply:
Well, on my machine, the command won’t succeed unless I run the command `reg import` as an administrator. Maybe I miss something.

Would you please try to run the command as a regular user, and then tell me if you manage to add the context menu?

Reply:
The [official documentation](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/reg-import) says:

> Copies the contents of a file that contains exported registry subkeys, entries, and values into the registry of the **local computer**.

To be honest, I copy-paste-adapted the logic from [Sublime Text](https://github.com/ScoopInstaller/Extras/blob/master/bucket/sublime-text.json) and [Notepad++](https://github.com/ScoopInstaller/Extras/blob/master/bucket/notepadplusplus.json).

Reply:
Oh, I think I got it: https://stackoverflow.com/questions/59917614/edit-the-windows-explorer-context-menu-for-only-a-single-user

I’ll try that out.

Reply:
Commit [8fe7396](https://github.com/ScoopInstaller/Extras/pull/8637/commits/8fe7396df251f6015db795000c8b29497dccd459) should do the trick.